src/lib/support/logging/Constants.h

```diff
@@ -63,6 +63,169 @@ enum LogModule
     kLogModule_Max
 };
 
+/* Log modules enablers. Those definitions can be overwritten with 0 to disable
```

Comment:
Yeah, but you would need to do that by defining all possible log categories for this module 0. Actually I just realized that in the same way we can disable global log categories and nobody uses _CHIP_USE_LOGGING, which is quite weird for me.

Reply:
Ok, but it would simplify the configuration architecture. At least I think these macros should be taken into account in `IsModuleCategoryEnabled` so that one can easily cut off the code for log generation.